### PR TITLE
Categories + Products Branch

### PR DESCRIPTION
**Pull request by tejas-muthya37**
This branch has the screens for the Categories and Products pages for my E-Commerce App.

### CODE REVIEW

**Review by mohibkay — APPROVED** [review]

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+.category-card:nth-child(3) {
+    text-shadow: 2px 2px skyblue;
+}
+
+.category-card:nth-child(4) {
+    text-shadow: 2px 2px #ff5f77;
+}
+
```
Comment (romabulani) [inline styles.css]: If possible, try to use variables for colors

File: Card/card.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+.button-group {
+    display: flex;
+    flex-direction: column;
+    gap: 0.75rem;
+    margin-top: 1rem;
+}
+
+.button-group button {
+    height: 40px;
```
Comment (romabulani) [inline Card/card.css:44]: Check if you could use rem here

**Review by romabulani — APPROVED** [review]
In product.html file, alt attribute for all the img tags are empty, please provide the values to it..

Rest all looks good to merge

File: Card/card.css (hunk 2 above)
Comment (tejas-muthya37) [reply Card/card.css:44]: Done. Thanks, all the suggestions incorporated.

**Pull request merged**